agent/api/task.go

```diff
@@ -118,15 +119,19 @@ type Task struct {
 	// used to look up the credentials for task in the credentials manager
 	credentialsID     string
 	credentialsIDLock sync.RWMutex
+
+	// memoryCPULimits to determine if task supports CPU, memory limits
+	memoryCPULimits     bool
```

Comment:
This doesn't come to me as a bool variable. May be names like `memoryCPULimitsSet` work?

Reply:
Maybe `memoryCPULimitsEnabled`

Reply:
'enabled' is what we use in the config variables.